nav2_waypoint_follower/include/nav2_waypoint_follower/plugins/wait_at_waypoint_arrival.hpp

```diff
@@ -0,0 +1,98 @@
+/*
+ * Software License Agreement (BSD License)
```

Comment:
Please change the licenses to Apache 2.0. The package is all apache 2.0 so if you made modifications to it, please use the same license

Reply:
As well in all files